db/migrate/20220914121554_add_eventable_fields_to_timeline_event.rb

```diff
@@ -0,0 +1,10 @@
+class AddEventableFieldsToTimelineEvent < ActiveRecord::Migration[7.0]
+  def change
+    change_table :timeline_events, bulk: true do |t|
+      t.integer :eventable_id
+      t.index :eventable_id
+      t.string :eventable_type
+      t.index :eventable_type
```

Comment:
I think there's a way to use `t.references` in the migration to avoid needing to add the columns manually, but I don't feel too strongly on that.